tests/ui/impl-trait/in-trait/success.rs

```diff
@@ -1,6 +1,6 @@
 // check-pass
 
-#![feature(lint_reasons)]
+#![cfg_attr(bootstrap, feature(lint_reasons))]
```

Comment:
you can just remove these attributes from tests. We don't run ui tests on stage 0